lib/new_relic/agent/instrumentation/rails3/action_controller.rb

```diff
@@ -127,7 +127,7 @@ def render_with_newrelic(*args, &block)
   # We can't be sure that this will work with future versions of Rails 3.
   # Currently enabled for Rails 3.1 and 3.2
   depends_on do
-    defined?(::Rails) && ::Rails::VERSION::MAJOR.to_i == 3 && ([1,2].member?(::Rails::VERSION::MINOR.to_i))
+    defined?(::Rails::VERSION) && ::Rails::VERSION::MAJOR.to_i == 3 && ([1,2].member?(::Rails::VERSION::MINOR.to_i))
```

Comment:
Should it be defined?(::Rails::VERSION::MAJOR) && defined?(::Rails::VERSION::MINOR) && ... ?

Reply:
That would be even safer, but not necessary imho.